Added schematic image instead of crude drawing in Planes tutorial

rakaar opened the pull request:
In chapter 1, planes tutorial
There was a schematic diagram of Gravitational lensing drawn using ASCII characters, which is actually like this
![Screenshot from 2020-12-24 15-23-53](https://user-images.githubusercontent.com/44907757/103081605-4ebdbf80-45fe-11eb-843c-b9274bd8b093.png)

But when it is run, it looks like this(different from original and difficult to understand)
![Screenshot from 2020-12-24 15-24-00](https://user-images.githubusercontent.com/44907757/103081642-5e3d0880-45fe-11eb-9747-684b2bac5abe.png)

So, I thought it would be better if there was an actual image, which would be clear. Added [this](https://lh3.googleusercontent.com/proxy/P8sLJaVKAQIxJPzl5UD_rLKW2_aXg8TTLREDU-DnpB8A9PQ2ICvft9hy4PtGQhx7oZXL-7G_ROpG_xAPIL4LCPxB1_zI6JfPpphDqtfvbLZd04MKtwdax_ei-Sw6568) image in the tutorial and changed the text above it to match.
Currently it looks like this...
![image](https://user-images.githubusercontent.com/44907757/103083273-03a5ab80-4602-11eb-9dda-567211ee271a.png)



I also added `.ipynb_checkpoints` in `.gitignore`, so that they are not tracked accidentally in someone's future PRs or commits. 
P.S The tutorials are great, really well written! Thanks for them!

rakaar committed:
Added schematic image instead of crude drawing in Planes tutorial notebook

Jammy2211 commented:
Thanks for the kind words about the tutorials - a lot of time and heart went into them! :)

Whilst I was super proud of my ASCII art skills at the time, removing this crude drawing of a strong lens is a good shout. I have one suggestion before accepting, can you change the label on the lens galaxy from 'gravitational source' to 'gravitational lens' and call the background 'Object' the 'Source'. The word 'source' is typically used to describe the background object being lensed, and so labeling the lens galaxy with the word source is a bit confusing (I know what you meant).

rakaar commented:
@Jammy2211 Changed as you had suggested
![image](https://user-images.githubusercontent.com/44907757/103085013-79ac1180-4606-11eb-9433-56665e4ce714.png)

You were right, the word `Gravitational Source` might be confused with the `source galaxy(s)`. Changed the labels as per the old ASCII drawing. I have also squashed the commits.

Jammy2211 commented:
Looks great, thanks! I'll rebuild the docs as soon as I wrap up this refactor :)
